src/modules/utils/directives/validators/Validate.js

```diff
@@ -77,19 +81,21 @@
                         }
 
                         _addInputPattern(pattern) {
-                            const create = () => {
-                                const patternName = Validate._getAttrName('pattern');
-                                if (!$attrs[patternName]) {
-                                    $attrs[patternName] = pattern;
-                                    this._createValidator('pattern');
-                                }
-                            };
+                            return new Promise((resolve) => {
+                                const create = () => {
+                                    const patternName = Validate._getAttrName('pattern');
+                                    if (!$attrs[patternName]) {
```

Comment:
In case `attrs[patternName]` already exists promise will never get resolved or rejected.